apps/backend/src/adoptions/adoptions.controller.ts

```diff
@@ -15,29 +16,34 @@ import {
 } from '@nestjs/common';
 import { JoiValidationPipe } from '../pipes/joi-validation-pipe';
 import { AdoptionsService } from './adoptions.service';
+import { AccessTokenGuard } from 'src/guards/accessToken.guard';
```

Comment:
```suggestion
import { AccessTokenGuard } from '../guards/accessToken.guard';
```

Reply:
Hey @belgamo Thanks for suggentions. I'm getting typescript error for this import. Should I ignore this ? 
Cannot find module '../src/guards/accessToken.guard' or its corresponding type

Reply:
Please try `../guards/accessToken.guard` without the `src`

Reply:
@belgamo  I fixed the import path. This import was also failing e2e specs.